packages/components/src/presentation-components/hooks/UseLatest.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useEffect, useRef } from "react";
+
+/** @packageDocumentation
+ * @module Core
+ */
+
+/**
+ * Hook for creating a reference that uses the latest passed value.
+ * @internal
+ */
+export function useLatest<T>(value: T) {
```

Comment:
I don't think we need to move this into separate file. It is used only in one place.

Reply:
Moved it back.